### PR TITLE
Adjust English in README

### PR DESCRIPTION
**Pull request by sxctic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ A Python library to control ffmepg from asyncio for [Home Assistant](https://www
 - Analyse a video/audio stream for noise or motion detection
 - Grab image from a stream
 
-Be carfull that you protect function calls to this library with `asyncio.shield`.
+Be careful that you protect function calls to this library with `asyncio.shield`.
```
Comment (MartinHjelmare) [inline README.md:9]: The English of this sentence is still not correct. I'll check with some people if we know the intention of this sentence.

Comment (MartinHjelmare) [reply README.md:9]:
```suggestion
Take care to protect function calls to this library with `asyncio.shield`.
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (sxctic) [reply README.md]: Yes, the better way to phrase that.

**Pull request merged**